container-toolkit/supported-platforms.md

```diff
@@ -9,28 +9,44 @@
 
 ## Linux Distributions
 
-Supported Linux distributions are listed below:
-
-| OS Name / Version        | amd64 / x86_64 | ppc64le | arm64 / aarch64 |
-| ------------------------ | -------------- | ------- | --------------- |
-| Amazon Linux 2023        | X              |         | X {sup}`1`      |
-| Amazon Linux 2           | X              |         | X               |
-| Open Suse/SLES 15.x      | X              |         |                 |
-| Debian Linux 10          | X              |         |                 |
-| Debian Linux 11          | X              |         |                 |
-| Centos 7                 | X              | X       |                 |
-| Centos 8                 | X              | X       | X               |
-| RHEL 7.x                 | X              | X       |                 |
-| RHEL 8.x                 | X              | X       | X               |
-| RHEL 9.x                 | X              | X       | X               |
-| Ubuntu 18.04             | X              | X       | X               |
-| Ubuntu 20.04             | X              | X       | X               |
-| Ubuntu 22.04             | X              | X       | X               |
-| Ubuntu 24.04             | X              |         | X               |
-
-The `arm64` / `aarch64` architecture includes support for Tegra-based systems.
-
-1. For Amazon Linux 2023 on Arm64, a `g5g.2xlarge` Amazon EC2 instance was used for validation.
+Releases are qualification-tested and expected to work on these platforms:
```

Comment:
I'm not sure what qualification-testing means. Is it generic enough that readers would know what it means? Are there qualifications that we test that are specific to the software that we can link to?

I'd be a little cautious with removing the word "support" entirely here. B/c I don't want this to read like "we support everything, everywhere, all the time". Is there an SLA on support and what the community can reasonably expect in terms of getting questions answered for unsupported use cases?

Reply:
> I'm not sure what qualification-testing means.

Qualification-testing IMO in software typically refers to a quality assurance stage as part of a release process, trying to make (pretty damn) sure that things really work, in addition to, say, CI tests.

Other than that, I allow myself to make a bit of a joke here and refer to NASA:

![image](https://github.com/user-attachments/assets/f036153b-f747-4c30-957c-ce36ba9fc6a0)

I think you inspire me to simplify. I will remove the "qualification-". Simplicity wins, so I will push a commit changing this to:

> Releases are tested and expected to work on these platforms